test/jest/Bladeburner/BladeburnerConsole.test.ts

```diff
@@ -0,0 +1,92 @@
+import type { Bladeburner } from "../../../src/Bladeburner/Bladeburner";
+import { Player, setPlayer } from "@player";
+import { PlayerObject } from "../../../src/PersonObjects/Player/PlayerObject";
+import { BladeburnerContractName } from "@enums";
+import { Contract } from "../../../src/Bladeburner/Actions";
+
+describe("Bladeburner Console", () => {
+  let inst: Bladeburner;
+  const FEEDBACK_CMDS = [["skill"], ["start"], ["help"], ["log"], ["automate"]];
```

Comment:
Why are these all single-element lists? I'm pretty sure if `each` takes a single argument, they don't need to be wrapped like this.

Reply:
Hey I wasn't aware of that. Cool!